distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -257,9 +257,8 @@ type Table
          - If duplicate columns, names or indices are provided, a
            `Duplicate_Column_Selectors`.
          - If a column index is out of range, a `Column_Indexes_Out_Of_Range`.
-         - If two distinct indices would refer to the same column, a
-           `Input_Indices_Already_Matched`, indicating that the additional
-           indices will not introduce additional columns.
+         - If two distinct indices refer to the same column, an
+           `Input_Indices_Already_Matched`.
```

Comment:
Why make the description less detailed? It is not that clear that the additional indices will not introduce additional columns, is it?

Reply:
Add a bit more description - tidying up the comment in line with grammarly suggestions.